src/main/java/org/jetbrains/research/kotlincodesmelldetector/reporting/AnonymousFeedback.java

```diff
@@ -0,0 +1,188 @@
+package org.jetbrains.research.kotlincodesmelldetector.reporting;
+
+import com.intellij.openapi.diagnostic.SubmittedReportInfo;
+import com.intellij.openapi.diagnostic.SubmittedReportInfo.SubmissionStatus;
+import org.eclipse.egit.github.core.Issue;
+import org.eclipse.egit.github.core.Label;
+import org.eclipse.egit.github.core.RepositoryId;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.client.PageIterator;
+import org.eclipse.egit.github.core.service.IssueService;
+import org.jetbrains.research.kotlincodesmelldetector.KotlinCodeSmellDetectorBundle;
+
+import javax.annotation.Nullable;
+import java.util.*;
+
+import static org.jetbrains.research.kotlincodesmelldetector.reporting.InformationType.*;
+
+/**
+ * Provides functionality to create and send GitHub issues when an exception is thrown by a plugin.
+ */
+class AnonymousFeedback {
```

Comment:
Please, let's do not copy-paste code which we do not need for now :)

Reply:
Ok, removed